### PR TITLE
Add missing placeholder icons

### PR DESCRIPTION
**Pull request by ferferga**
Add placeholder icons for collections and playlists that doesn't have images

### CODE REVIEW

File: src/components/cardbuilder/cardBuilder.js [hunk 1]
```diff
@@ -1547,6 +1547,10 @@ import 'programStyles';
                     return '<span class="cardImageIcon material-icons book"></span>';
                 case 'Folder':
                     return '<span class="cardImageIcon material-icons folder"></span>';
+                case 'BoxSet':
+                    return '<span class="cardImageIcon material-icons collections"></span>';
+                case 'Playlist':
+                    return '<span class="cardImageIcon material-icons queue_music"></span>';
```
Comment (dmitrylyzo) [inline src/components/cardbuilder/cardBuilder.js:1553]: Actually it could also be a video playlist. But music is used more often.

Comment (ferferga) [reply src/components/cardbuilder/cardBuilder.js:1553]: @dmitrylyzo Didn't found a more appropriate icon for it sadly. The queue or playlist ones doesn't look like related to playlists.

I checked material-icons.io for reference

Comment (MrTimscampi) [reply src/components/cardbuilder/cardBuilder.js:1553]: This one could fit: https://material.io/resources/icons/?icon=queue&style=baseline

Comment (dmitrylyzo) [reply src/components/cardbuilder/cardBuilder.js:1553]: list-2

Comment (MrTimscampi) [reply src/components/cardbuilder/cardBuilder.js:1553]: Nice, view_list would be perfect :)

It stays consistent as well.

Comment (dkanada) [reply src/components/cardbuilder/cardBuilder.js:1553]: I agree with the view list icon.

Comment (ferferga) [reply src/components/cardbuilder/cardBuilder.js]: @dkanada @MrTimscampi @dmitrylyzo Best one imo as well, so done!

**Pull request merged**